demos/browser/app/meeting/meeting.ts

```diff
@@ -550,7 +551,12 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver
   }
 
   async initializeMeetingSession(configuration: MeetingSessionConfiguration): Promise<void> {
-    const logger = new ConsoleLogger('SDK', LogLevel.DEBUG);
+    let logger: MeetingSessionPOSTLogger | ConsoleLogger = null;
+    if (!(DemoMeetingApp.BASE_URL.includes('localhost') || DemoMeetingApp.BASE_URL.includes("127.0.0.1"))) {
```

Comment:
https://github.com/aws/amazon-chime-sdk-js/pull/162 will overrride this `if` statement, but replace ```"``` to ```'``` when you resolve other conflicts.